openpmd_viewer/openpmd_timeseries/data_reader/h5py_reader/field_reader.py

```diff
@@ -25,9 +26,11 @@ def read_field_cartesian( filename, field_path, axis_labels,
     filename : string
        The absolute path to the HDF5 file
 
-    field_path : string
-       The relative path to the requested field, from the openPMD meshes path
-       (e.g. 'rho', 'E/z', 'B/x')
```

Comment:
`field_path` was split into `field` and `coord`, since, for the future `openpmd-api` implementation of this function, it would be more natural to have this 2 variables separate.